Mathlib/Topology/UnitInterval.lean

```diff
@@ -133,6 +141,58 @@ theorem strictAnti_symm : StrictAnti σ := fun _ _ h ↦ sub_lt_sub_left (α :=
 theorem half_le_symm_iff (t : I) : 1 / 2 ≤ (σ t : ℝ) ↔ (t : ℝ) ≤ 1 / 2 := by
   rw [coe_symm_eq, le_sub_iff_add_le, add_comm, ← le_sub_iff_add_le, sub_half]
 
+@[simp]
+lemma symm_eq_one_iff_eq_zero {i : I} : σ i = 1 ↔ i = 0 := by
```

Comment:
```suggestion
lemma symm_eq_one {i : I} : σ i = 1 ↔ i = 0 := by
```
The RHS is implied

Reply:
I changed it to `symm_eq_one_iff`. Are you fine with that as well?

Reply:
I would rather remove the `_iff`. It's clearly implied

Reply:
Alright, I removed it.